lib/generators/shopify_app/scopes_storage/templates/home_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  include ShopifyApp::EmbeddedApp
+  include ShopifyApp::RequireKnownShop
+  include ShopifyApp::ScopesVerification
```

Comment:
I believe templates work with erb templating code as well. Something like 
```suggestion
  <%= 'include ShopifyApp::ScopesVerification' if user_answered_yes_to_prompt? %>
```

Reply:
This variable should be configurable in the install generator class or the method itself that calls `template(src, dest)`

Reply:
Yeah that makes sense to me!